Configurable servers (#485)
Ouranosinc/raven: docs/source/dev_guide.rst

Running notebooks tests
-----------------------

> Assuming that the ``raven`` conda env has already been created and is up-to-date and `raven-wps`` has been installed with ``make develop``::

Reviewer: Missing a ``` ` ``` before `raven-wps`

Highlighting is better if using the following instead of `::`
```rst
.. code-block:: shell
  
    [...code...]

```

        # to test all notebooks
        make WPS_URL=https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps test-notebooks

    Or, alternatively:

    .. code-block:: console

        # to test a single notebook (note the .run at the end of the notebook path)
        make WPS_URL=https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps docs/source/notebooks/Subset_climate_data_over_watershed.ipynb.run

> We can also override all three of the server variables (``WPS_URL``, ``FINCH_WPS_URL``) to pick and choose any servers/services from anywhere we want.

Reviewer 2: ```suggestion
We can also override both of the server variables (``WPS_URL``, ``FINCH_WPS_URL``) to pick and choose any servers/services from anywhere we want.
```
